libMBIN/Source/NMS/GameComponents/GcRealityManagerData.cs

```diff
@@ -50,6 +50,10 @@ public class GcRealityManagerData : NMSTemplate
         /* 0x0818 */ public List<NMSString0x80> AlienPuzzlesTables;
         [NMS(Size = 0x65, EnumType = typeof(GcInteractionType.InteractionTypeEnum))]
         /* 0x0828 */ public bool[] LoopInteractionPuzzles;
+
+        //[NMS(Size = 0x3, Ignore = true)]
```

Comment:
why is this added but commented out?

Reply:
You said before it is not technically needed but since I don't know exactly how those rules work yet, I did try it out both ways (commented/commented out).  Works both ways.  Left it there as a reminder.

Reply:
maybe pick to keep it or not. Leaving it in as comment it pointless.
In this case, as you have noticed, it won't matter if it is there or not

Reply:
removed commented lines